lib/fake_stripe.rb

```diff
@@ -30,9 +30,11 @@ def self.reset
   end
 
   def self.stub_stripe
-    Stripe.api_key = 'FAKE_STRIPE_API_KEY'
-    FakeStripe.reset
-    stub_request(:any, /api.stripe.com/).to_rack(FakeStripe::StubApp)
+    if defined?(Stripe)
```

Comment:
The README's instructions say to call `FakeStripe.stub_stripe` in e.g. `spec_helper.rb`. If the `stripe` gem is not installed, calling this method will be a no-op (right?).

I think I'm missing something, and would be happy to have it explained to me, but it makes sense to me to make `stripe` a runtime dependency, and not change this method at all. Making this work without a `Stripe` constant defined is odd to me: why install `fake_stripe` if `stripe` isn't installed?